db/migration.manager.go

```diff
@@ -16,9 +15,20 @@ import (
 
 // RunMigrations ... Migrate all files in migrations package.
 func RunMigrations() (*mongo.Database, error) {
-	uri := fmt.Sprintf("mongodb://%s:%s@%s:%s", beego.AppConfig.String("mongo_user"), beego.AppConfig.String("mongo_pass"), beego.AppConfig.String("mongo_host"), "27017")
-	opt := options.Client().ApplyURI(uri)
-	client, err := mongo.NewClient(opt)
+	dbURL := beego.AppConfig.String("mongo_host")
+	dbPort := "27017"
```

Comment:
Traer este valor de una variable de entorno. En `conf/app.conf` agregar la siguiente línea:
```ini
mongo_port = ${CUENTAS_CONTABLES_CRUD_DB_PORT||27017}
```
Lo anterior permite tomar el 27017 como valor por defecto en caso de no estar `CUENTAS_CONTABLES_CRUD_DB_PORT` declarada, y también que se pueda aplicar el siguiente _suggested change_:
```suggestion
	dbPort := beego.AppConfig.String("mongo_port")
```